### PR TITLE
Upgrade to Meta Package Manager plugin v3.4.0.

### PR DESCRIPTION
**Pull request by kdeldycke**
In sync with the [new `mpm` 3.4.0 release](https://github.com/kdeldycke/meta-package-manager/releases/tag/v3.4.0), this is the brand new BitBar plugin.

Changelog:
* Forces syncing of local package manager database on refresh.
* Allow switching of rendering mode from flat to sub-menu by the way of the `BITBAR_MPM_SUBMENU` environment variable.
* Add minimal requirement on `meta-package-manager` module in the invoked `pip` command.
* Each update line can be called with the `option` modifier to run the CLI into a terminal. Useful for debugging.
* Code linted by Black formatter.

### CODE REVIEW

File: Dev/MetaPackageManager/meta_package_manager.7h.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # <bitbar.title>Meta Package Manager</bitbar.title>
-# <bitbar.version>v2.5.0</bitbar.version>
+# <bitbar.version>v3.4.0</bitbar.version>
 # <bitbar.author>Kevin Deldycke</bitbar.author>
 # <bitbar.author.github>kdeldycke</bitbar.author.github>
 # <bitbar.desc>List outdated packages and manage upgrades.</bitbar.desc>
```
Comment (matryer) [inline Dev/MetaPackageManager/meta_package_manager.7h.py:7]: Any chance you could add an image too?

Comment (kdeldycke) [reply Dev/MetaPackageManager/meta_package_manager.7h.py:7]: There's already an image! :) See: https://github.com/matryer/bitbar-plugins/pull/1479/commits/501efb61873bbc8bed071dd612c162e934091bd5#diff-35b8b72745b2f4b47516a8ceb1054c36f61ba6cbe877bd02a70af9b08f835aaeR9

Comment (kdeldycke) [reply Dev/MetaPackageManager/meta_package_manager.7h.py:7]: Image worth 1000 words:

Screen Shot 2020-12-11 at 15 57 03

😉

**Comment by kdeldycke** [comment]
This PR has been replaced by https://github.com/matryer/xbar-plugins/pull/1613

**Pull request closed without merging**